index.html

```diff
@@ -0,0 +1,94 @@
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+  <header>
+    <nav id="navbar">
+      <ul class="nav-links">
+        <li><a href="#welcome-section" class="nav-link">Welcome</a></li>
+        <li><a href="#about" class="nav-link">About</a></li>
+        <li><a href="#projects" class="nav-link">Projects</a></li>
+        <li><a href="#contact" class="nav-link">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section id="welcome-section">
+    <h1>Hey, I am Hamasa Noorzai</h1>
+    <h2 class="note">A Web Developer</h2>
+    <p class="info">
+      I am Hamasa, a junior developer and BCS student at the American University of Afghanistan. Passionate about technology, I excel in programming, data structures, and software engineering. I apply my skills in HTML and CSS to real-world projects.
+    </p>
+  </section>
+  <section id="about">
+    <div class="container">
+      <div class="her-content">
+        <h2>Read about ME</h2>
+        <p>Hamasa is a junior developer and a Bachelor of Computer Science (BCS) student at the American University of Afghanistan (AUAF). Growing up in Kabul, Hamasa developed an early fascination with technology, which led to a keen interest in computer science. As a Junior Developer, Hamasa effectively applies theoretical knowledge to real-world projects, showcasing a strong problem-solving mindset and proficiency in HTML and CSS.</p>
+      </div>
+      <div class="her-image">
+        <img src="images/me.jpg" alt="my image">
+      </div>
+    </div>
+  </section>
+  <section id="projects" class="project-section">
+    <h1 class="project-section-header">These are some of my projects</h1>
+    <div class="project">
+      <a href="https://hamas2000.github.io/Survey-form/" class="project-tile" target="_blank" rel="noopener">
+        <img src="images/form-img.png" alt="survey-form">
+        <p class="project-tile">Survey Form</p>
+      </a>
+      <a href="https://hamas2000.github.io/tribute-page/" class="project-tile" target="_blank" rel="noopener">
+        <img src="images/tribute-img.png" alt="tribute-page">
+        <p class="project-tile">Tribute Page</p>
+      </a>
+      <a href="https://hamas2000.github.io/Product-Landing-Page/" class="project-tile" target="_blank" rel="noopener">
+        <img src="images/landing-page.png" alt="product-page">
+        <p class="project-tile">Product Landing Page</p>
+      </a>
+      <a href="https://hamas2000.github.io/-Technical-Documentation-Page/" class="project-tile" target="_blank" rel="noopener">
+        <img src="images/js.png" alt="-Technical-Documentation-Page">
+        <p class="project-tile">Technical Documentation Page</p>
+      </a>
+    </div>
+  </section>
```

Comment:
![image](https://github.com/Hamas2000/Personal-Portfolio/assets/96838030/e02252f2-7d01-47ef-b0e8-e837a1edacc5)

- For the desktop version everything is fine, you just need to add the SOURCE_CODE (your project github link) here beside your project deployed link.